charts/intel-gpu-resource-driver/templates/validating-admission-policy-binding.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: resourceslices-policy-dra-kubelet-plugin-gpu
```

Comment:
The resources 'validating admission policy' and 'validating admission policy binding' have the same names across all drivers, causing conflicts when installing multiple drivers with Helm in the same cluster. Renaming these resources to include the driver type could solve the issue

Reply:
I guess things that are common between drivers and just need a different name, could be put into a separate subchart (on which drivers depends on), but just having separate files for drivers looks fine too.